Opcodes/sfont.c

```diff
@@ -750,7 +775,7 @@ static int32_t SfPlayMono_set(CSOUND *csound, SFPLAYMONO *p)
             double freq, orgfreq;
             double tuneCorrection = split->coarseTune + layer->coarseTune +
               (split->fineTune + layer->fineTune)*0.01;
-            int32_t orgkey = split->overridingRootKey;
+            int32_t orgkey = split->overridingRootKey, notenum = notnum;
```

Comment:
notenum and notnum seem very close in naming and easy to have mistakes via typo. Could use a different variable name.

Reply:
ok